src/Redux/actionCreators/PaymentActions.js

```diff
@@ -36,6 +37,23 @@ export const executePayment = paymentExecObj => async () => {
   return await executePaymentAPI(token, paymentExecObj);
 };
 
-export const updatePaypalInProgress = (orderId, paymentId, paypalPaymentId, PayerID) => dispatch => {
-  dispatch({ type: UPDATE_PAYPAL_IN_PROGRESS, payload: { orderId, paymentId, paypalPaymentId, PayerID } });
+const orderDetailsAPI = (token, orderId) => {
+  const apiName = APIEndpoints.ORCHESTRATOR.name;
+  const apiPath = `${APIPaths.ORDER_DETAILS}/${orderId}`;
+  // const queryParams = { order_id: orderId };
```

Comment:
Can be deleted.

Reply:
cleaned it

Reply:
https://github.com/singnet/snet-dapp/pull/404/commits/fa4c00dc9951b199b291663830d71106dd382787